src/Stepper.js

```diff
@@ -62,7 +65,7 @@ class Stepper {
                 }
             }
 
-            doing(1 - easing(time));
+            doing(getNow(time));
```

Comment:
It is not intutive that conditions and operation code is put as parameter so i think that it is better to extract this code.